blog/tests/test_blog_models.py

```diff
@@ -190,10 +190,8 @@ def test_published(self):
         # self.assertEqual(len(mgr.published()), 50)
```

Comment:
tiun ĉi teston mi forkomentis ĉar kreado de 50 objektoj (eĉ per `build_batch` kaj `bulk_create` ) eeegeeee malrapidigis testojn. post via ŝanĝo ĝis des pli estas forigenda.